src/csharp/Microsoft.Spark/Sql/CastUnpickledItems.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Spark.Sql
+{
+    /// <summary>
+    /// Cast UnpickledItems.
+    /// </summary>
+    internal class CastUnpickledItems
```

Comment:
Can we make this class static?

Reply:
Yes, may I ask why this should be a static class?

Reply:
Is there any need to instantiate this class? If not, we should make it static.